actors/abi/sector.go

```diff
@@ -92,9 +92,9 @@ type PartialTicket []byte // 32 bytes
 type PoStVerifyInfo struct {
 	Randomness      PoStRandomness
 	CommR           cid.Cid
-	Candidates      []PoStCandidate      // From OnChain*PoStVerifyInfo
-	Proofs          []PoStProof          // From OnChain*PoStVerifyInfo
-	EligibleSectors map[SectorID]cid.Cid // TODO: HAMT?
+	Candidates      []PoStCandidate // From OnChain*PoStVerifyInfo
+	Proofs          []PoStProof     // From OnChain*PoStVerifyInfo
+	EligibleSectors cid.Cid
```

Comment:
You can't do thAT in this struct. Sorry, I should have been more direct that a HAMT cannot be used here. This structure is passed to the PoSt verification syscall and ultimately the proofs library, and must be fully self-contained.

I think it should be either a `[]SectorID` or, probably, a bitfield. You could do the former for expediency now, but `TODO bitfield`.